rewrite-gradle/src/main/java/org/openrewrite/gradle/UpdateJavaCompatibility.java

```diff
@@ -79,8 +81,15 @@ public class UpdateJavaCompatibility extends Recipe {
     @Nullable
     Boolean addIfMissing;
 
+    @Option(displayName = "Add Java toolchain if missing",
+            description = "Adds the specified Java toolchain if one is not found.",
+            required = false)
+    @Nullable
+    Boolean addToolchainIfMissing;
```

Comment:
Just an idea, but we could possibly use DeclarationStyle as the mechanism to describe how you as a user want the version to be defined. There's already the `addIfMissing` option, so that could work as a possibility.